src/components/BooksList.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Book from './Book';
+
+const BooksList = ({ books }) => (
```

Comment:
- [ ] Since BookList is considered a container component (components that are connected with Redux) it should be placed in `src/containers` directory.
![Screenshot from 2021-03-12 13-15-13](https://user-images.githubusercontent.com/30977845/110967317-011b3e00-8335-11eb-85fb-75d0c34df02a.png)